### PR TITLE
1028 layout useful

### PR DESCRIPTION
**Pull request by docwilmot**
Fixes https://github.com/backdrop/backdrop-issues/issues/1028
Replaces https://github.com/backdrop/backdrop/issues/1341

### CODE REVIEW

File: core/modules/layout/css/layout.admin.css [hunk 1]
```diff
@@ -12,6 +12,41 @@
...
+    display: inline;
+  }
+}
+
+#layout-list .layout-conditions-list ul {
+  margin-left: 0;
+}
+
+#layout-list .item-list ul li {
+  list-style-type: square;
```
Comment (klonos) [inline core/modules/layout/css/layout.admin.css:43]: Why square list icons? I think that our seven style guide dictates using the default.

**Comment by klonos** [comment]
...other than that, looking good :+1:

File: core/modules/layout/css/layout.admin.css (hunk 1 above)
Comment (docwilmot) [inline core/modules/layout/css/layout.admin.css:43]: I hadnt considered the official style guide, just thought square bullets matched the template icons better. Will change them back.

**Comment by jenlampton** [comment]
does this need a reroll? or, retest?

**Comment by quicksketch** [comment]
I'll take a stab at rerolling this.

**Comment by jenlampton** [comment]
Oops @quicksketch I'm almost done.

**Comment by jenlampton** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/1385

**Pull request closed without merging**